doc/nrf/links.txt

```diff
@@ -517,6 +522,8 @@
 
 .. ### Source: docs.nordicsemi.com
 
+.. _`Nordic Semiconductor TechDocs`: https://docs.nordicsemi.com
+
```

Comment:
Can we also add a link to Amazon github page under the  `Source: github.com` section?

```
.. _`Amazon Sidewalk Sample IoT App`: https://github.com/aws-samples/aws-iot-core-for-amazon-sidewalk-sample-app
```